### ISSUE TITLE
Rephase addon installation prompt to make it more clear.

### ISSUE THREAD

**Issue by karandatwani92** [Laravel-Backpack/CRUD#5129]
![Screenshot 2023-06-09 at 8 15 43 PM](https://github.com/Laravel-Backpack/CRUD/assets/8214221/a32cdbe5-b68b-4e82-b659-4aebb12ed848)

@tabacitu and I discussed this and decided to rephrase it. So let's do this **tiny** change.

### What are we trying to do?
Generally, users see the last line only of the prompt and ignore's above-created texts.
So, When we are asking them to enter an option number, some users may get confused about what option no.

### Solution
Change text to: **enter an option number from above**.

@tabacitu is it OK or we write something else?

**Comment by fronbow** [comment]
I would rephrase it to "Enter an option number from above or any other key to continue installation" or somesuch

**Comment by tabacitu** [comment]
> I would rephrase it to "Enter an option number from above or any other key to continue installation" or somesuch

☝️

**Comment by pxpm** [comment]
Fixed by https://github.com/Laravel-Backpack/CRUD/pull/5617
